### DISCUSSION TITLE
Hello from ImagePainting

### DISCUSSION THREAD

**Discussion by jbreizh** [MartinNohr/MagicImageWand#66] [Ideas]
Hello,
same topic different way and different board choice (ESP8266) for [ImagePainting](https://github.com/jbreizh/ImagePainting3)
Maybe you can find some idea from my code
Have a nice day

**Comment by MartinNohr** [comment]
Your hardware is much simpler than mine! But of course, I do all the work in the esp32 without using a phone.
There are at least two things that I like about your design. 1. The "cut" feature to add black lines in the image. 2. The ability to wait for a key when running a playlist (what I call a macro on my system).
Thanks for sharing. If you don't mind I think I will add both 1. and 2. to my system.
I just added a "crop" feature to mine that lets you select which part of the image you want to display on the LED's. You might consider adding that to yours if you haven't done that already.

**Comment by MartinNohr** [comment]
BTW, I didn't want to require the user to have a cell phone or other computer in order to run the wand, that is why I made the hardware more complex and used the more powerful ESP32 with a display. The trade-off is course that it is much harder to build. It is pretty easy with my PCB, but then you need to get a copy of that from me. So for ease of building, yours is very nice. Thanks again for sharing. I appreciate it.
